.vscode/launch.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
I don't think it makes sense to include this task here, as most users won't have edge installed, and neither do I think we want to explain to people how to set up the debugger to work in this case. Let's keep the tutorial simple 🙂